packages/reporter/src/errors/test-error.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react'
+import FlashOnClick from '../lib/flash-on-click'
+
+class TestError extends Component {
```

Comment:
Since this is stateless, I'd prefer to have it be a functional component instead of a class.